docs/node/week-1/lesson.md

```diff
@@ -77,59 +77,74 @@ So what's the big deal? I can see this information on web sites already!
 
 What's an API and how is it different from a web site?
 
-## 2) Intro to Node on Glitch
+## 2) Introduction to Node
 
-### 2.1) Running a Simple Server
+### 2.1) Investigating a Simple Server
 
-Run the simplest web server code.
+Let's investigate a really simple web server made in Node. You can check it out running here:
```

Comment:
```suggestion
Let's investigate a web server made in Node. You can check it out running here:
```

Avoid saying "simple" and "just" - it's intended to put people at ease but it counter-intuitively makes people feel incompetent when learning a new thing, to be told it's simple. I understand in this case the server is unfortunately called simple server.